### PR TITLE
nits: more ics29 nits

### PR DESCRIPTION
**Pull request by seantking**
## Description

More nits from audit

closes: #XXXX

- [x] Nit: no caps in errors
- [x] Nit: godoc missing for Validate() in types/msgs.go
- [x] Nit: godoc missing for NewIdentifiedPacketFee
- [x] Use strings.TrimSpace for MsgRegisterCounterpartyAddress

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/29-fee/types/errors.go [hunk 1]
```diff
@@ -14,4 +14,5 @@ var (
 	ErrCounterpartyAddressEmpty      = sdkerrors.Register(ModuleName, 7, "counterparty address must not be empty")
 	ErrForwardRelayerAddressNotFound = sdkerrors.Register(ModuleName, 8, "forward relayer address not found")
 	ErrFeeNotEnabled                 = sdkerrors.Register(ModuleName, 9, "fee module is not enabled for this channel. If this error occurs after channel setup, fee module may not be enabled")
+	ErrInvalidFee                    = sdkerrors.Register(ModuleName, 10, "one or more fees are invalid")
```
Comment (seantking) [inline modules/apps/29-fee/types/errors.go:17]: Unsure about the error message here, please feel free to suggest something else.

File: modules/apps/29-fee/types/msgs.go [hunk 2]
```diff
@@ -28,7 +30,7 @@ func (msg MsgRegisterCounterpartyAddress) ValidateBasic() error {
 		return sdkerrors.Wrap(err, "failed to convert msg.Address into sdk.AccAddress")
 	}
 
-	if msg.CounterpartyAddress == "" {
+	if strings.TrimSpace(msg.CounterpartyAddress) == "" {
```
Comment (damiannolan) [inline modules/apps/29-fee/types/msgs.go:33]: Worth adding an additional test case with a whitespaced empty string

File: modules/apps/29-fee/types/msgs.go [hunk 3]
```diff
@@ -82,12 +84,12 @@ func (msg MsgPayPacketFee) ValidateBasic() error {
 
 	// if any of the fee's are invalid return an error
 	if !msg.Fee.AckFee.IsValid() || !msg.Fee.ReceiveFee.IsValid() || !msg.Fee.TimeoutFee.IsValid() {
-		return sdkerrors.ErrInvalidCoins
+		return ErrInvalidFee
```
Comment (damiannolan) [inline modules/apps/29-fee/types/msgs.go:87]: Is there a need to add a new error type? What about wrapping the `ErrInvalidCoins` with additional info? 

eg. `sdkerrors.Wrap(sdkerrors.ErrInvalidCoins, "incentivized packet contains one or more invalid fees")`

Comment (colin-axner) [reply modules/apps/29-fee/types/msgs.go:87]: I like @damiannolan suggestion

**Review by colin-axner — APPROVED** [review]
ACK, I agree with @damiannolan's review comments

File: modules/apps/29-fee/types/msgs.go (hunk 2 above)
Comment (seantking) [reply modules/apps/29-fee/types/msgs.go:33]: done
